lib/opsgenie_alert_tagging.rb

```diff
@@ -0,0 +1,78 @@
+require 'httparty'
+require 'dotenv/load'
+require 'pry'
+
+class OpsgenieAlertTagging
+
+  attr_reader :date
+
+  def initialize(date)
+    @date = date
+  end
+
+  def call
+    update_opsgenie
+  end
+
+  def list_all_alerts
+    parse_date = Date.parse(date)
+    format_date = parse_date.strftime("%d-%m-%Y")
+
+    results = HTTParty.get("https://api.opsgenie.com/v2/alerts?query=createdAt%3A#{format_date}&sort=createdAt&order=desc", :headers => {
```

Comment:
The JSON this returns is limited to 20 results its would seem. it does however provide pagination details.
```
"paging":{"next":"https://api.opsgenie.com/v2/alerts?limit=20&sort=createdAt&offset=20&order=desc&query=createdAt%3A24-02-2020","first":"https://api.opsgenie.com/v2/alerts?limit=20&sort=createdAt&offset=0&order=desc&query=createdAt%3A24-02-2020","last":"https://api.opsgenie.com/v2/alerts?limit=20&sort=createdAt&offset=160&order=desc&query=createdAt%3A24-02-2020"}
```

It would seem it limits the limit to 100 items.